backend/canisters/storage_index/impl/src/lifecycle/post_upgrade.rs

```diff
@@ -27,4 +27,10 @@ fn post_upgrade(args: Args) {
     init_state(env, data, args.wasm_version);
 
     info!(version = %args.wasm_version, "Post-upgrade complete");
+
+    mutate_state(|state| {
```

Comment:
Which users don't have this enabled currently?

Reply:
It looks like we never enabled it for existing users. So it is set to false for users who registered before Feb 2023.